### ISSUE TITLE
Bump dependency to 1.19 and upgrade client to accept Context

### ISSUE THREAD

**Issue by Jeffwan** [kubeflow/common#125]
There's a dependency to accept https://github.com/kubeflow/common/pull/123

Major reason is 
1. kubeflow/common uses 1.16 and volcano latest api repo uses 1.19. 
2. client-go accepts context in 1.18. https://v1-18.docs.kubernetes.io/docs/setup/release/notes/#client-go

We will need to upgrade k8s dependency to 1.19 directly and update clients to accept `Context.TODO()` and this will unblock volcano api changes

/cc @Thor-wl @shinytang6

**Comment by Jeffwan** [comment]
We solve dependency issue by importing a volcano api version with lower k8s version 1.16.

Then we don't have to upgrade k8s dependency that urgent. Bumping to 1.19 or even newer version can be postponed

**Comment by Jeffwan** [comment]
https://github.com/kubeflow/common/pull/126 resolve this issue
